rocks-bin/src/remove.rs

```diff
@@ -30,9 +30,9 @@ pub async fn remove(remove_args: Remove, config: Config) -> Result<()> {
         &RemotePackage::new(remove_args.name.clone(), target_version.clone()).into_package_req(),
     ) {
         Some(package) => Ok(rocks_lib::operations::remove(
-            package,
```

Comment:
suggestion: let's change the CLI to take a non-empty list of packages.

Reply:
sounds good, on it

Reply:
That's fine. I guess you  can pass in `foo@1.0 foo@2.0`.
We should filter out (and warn if found) any packages that are dependencies of other packages which aren't to be deleted, but that's out of scope for this PR.

Reply:
Yep, we want to check to see if the package's id exists in the `entrypoints` table before deleting it.

jujutsu uses a special selector syntax whenever you want to operate on multiple values: `all:the-thing`. I find that interesting and maybe we could borrow that for our own delete command (e.g. `rocks remove all:neorg`).

Reply:
Hmm, wouldn't that be the same as `neorg@*`?

Reply:
Not quite: `all:` says that you accept the fact that you're selecting many versions. `neorg@*` or something like `neorg>=7.0.0` with two versions of `neorg` installed would both error (as it could be a potentially unintended action). You'd have to do `all:neorg>=7.0.0` (borrowing jj's syntax here) to make it work.

Reply:
I see. Sure, why not?